custom-steps/ffmpeg/post-build.ps1

```diff
@@ -0,0 +1,92 @@
+param (
+    [Parameter(Mandatory=$true)][string]$BuildArtifactsPath
+)
+
+. "$PSScriptRoot\..\..\util.ps1"
+
+$IsOnMacOS = Check-IsOnMacOS
+
+if(-not $IsOnMacOS) {
+    Write-Host "Not on Mac OS.  Exiting..."
```

Comment:
Same comment as earlier. Indicating why, and whether this is an acceptable thing or not is probably more relevant to whoever is reading this output.

Reply:
Fixed: https://github.com/TechSmith/ThirdParty-Packages-vcpkg/pull/1/commits/e19be2ec96316ea9170742f02b82dcb7bd4ca854